adds 24 pin FPC symbol and footprint

lewispg228 opened the pull request.

nseidle commented:
I touched up the symbol. Looks fine. I added the pitch to the part name. Additionally, we should consider calling out in the symbol that it is bottom contact only (I added to description but perhaps more).

The FP has issues. REF* is on the silk layer. The multiple overlapping mount pads should probably be united as a single polygon. They should be NCs. I'm not sure I trust this FP - please double check it against the datasheet or let me know if it's been used well on a proto.

lewispg228 commented:
Hey @nseidle , Sorry for the delay here. This is ready for another review please!

I corrected the mount pads to graphical shapes, and moved Ref** text to F.Fab layer.

Also, I double-checked this footprint against the datasheet. FWW, I originally imported this from a known-good footprint in the old Eagle Library. It was used on the CAMERA-HIMAX device on the [SparkFun Edge Development Board - Apollo3 Blue](https://www.sparkfun.com/products/15170).

nseidle commented:
Looks good. I renamed things to match library defaults. I also added the dual contact 24 pin 0.5mm (that you may want to consider = cheaper).
